packages/plugins/users-permissions/admin/src/pages/Roles/EditPage/tests/index.test.js

```diff
@@ -111,46 +140,497 @@ it('renders users-permissions edit role and matches snapshot', () => {
+                    <div
+                      class="c1 "
+                    >
+                      <div
+                        class="c20"
+                      >
+                        <div>
+                          <div
+                            class="c21"
+                          >
+                            <div
+                              class="c1 c4"
+                            >
+                              <label
+                                class="c9 c10"
+                                for="textinput-1"
+                              >
+                                Name
+                              </label>
+                            </div>
+                            <div
+                              class="c1 c3 c22"
+                            >
+                              <input
+                                aria-invalid="false"
+                                class="c23"
+                                id="textinput-1"
+                                name="name"
+                                value="Authenticated"
+                              />
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div
+                    class="c19"
+                  >
+                    <div
+                      class="c1 "
+                    >
+                      <div
+                        class="c24"
+                      >
+                        <div>
+                          <div
+                            class="c21"
+                          >
+                            <div
+                              class="c1 c4"
+                            >
+                              <label
+                                class="c9 c10"
+                                for="textarea-2"
+                              >
+                                Description
+                              </label>
+                            </div>
+                            <textarea
+                              aria-invalid="false"
+                              class="c25"
+                              id="textarea-2"
+                              name="description"
+                            >
+                              Default role given to authenticated user.
+                            </textarea>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
               </div>
             </div>
-            <p
-              class="c5 c6"
-            >
-              Default role given to authenticated user.
-            </p>
           </div>
         </div>
       </form>
     </main>
   `);
 });
+
+it("can edit a users-permissions role's name and description", async () => {
```

Comment:
Maybe you can add to test the errors to make sure that the app doesn't break with formatMessage